easybuild/easyconfigs/c/CrayGNU/CrayGNU-2016.03.eb

```diff
@@ -0,0 +1,19 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayGNU'
+version = '2016.03'
```

Comment:
@jgphpc I'd prefer is this is added via it's own PR, with an HPL easyconfig accompanying it

c @gppezzi